src/main/java/com/std/tothebook/exception/GlobalExceptionHandler.java

```diff
@@ -1,19 +1,24 @@
 package com.std.tothebook.exception;
 
 import com.std.tothebook.exception.dto.ErrorResponse;
+import lombok.extern.slf4j.Slf4j;
 import org.springframework.http.ResponseEntity;
 
 import org.springframework.util.StringUtils;
 import org.springframework.web.bind.annotation.ExceptionHandler;
 import org.springframework.web.bind.annotation.RestControllerAdvice;
 
+@Slf4j
 @RestControllerAdvice
 public class GlobalExceptionHandler {
 
     @ExceptionHandler(ExpectedException.class)
     public ResponseEntity<ErrorResponse> handleCustomException(ExpectedException e) {
-        // TODO log
-        System.out.println(e.toString());
+        this.log(e);
```

Comment:
```suggestion
        log(e);
```
this를 쓴 이유가 있을까요?
그냥 호출해도 큰 차이가 없다고 생각합니다.

확인 완료했습니다.